service_container/debug.rst

```diff
@@ -21,6 +21,21 @@ To see a list of all of the available types that can be used for autowiring, run
 
     $ php bin/console debug:autowiring
 
+How to Debug Service Tags & List Tagged Services
+------------------------------------
+You can find out what services are tagged with a specific tag using the following command:
+
+.. code-block:: terminal
+
+    $ php bin/console debug:container --tag=kernel.event_listener
+
+Partial search is also available:
```

Comment:
Perhaps document that it will prompt user with available matching tags

Reply:
Done.